scripts/self-signed-ca/conf/client.conf

```diff
@@ -36,4 +36,3 @@ nsCertType             = client
 keyUsage               = critical, nonRepudiation, digitalSignature, keyEncipherment
 extendedKeyUsage       = critical, clientAuth, codeSigning
 subjectKeyIdentifier   = hash
-authorityKeyIdentifier = keyid, issuer
```

Comment:
https://github.com/openssl/openssl/issues/22966

`authorityKeyIdentifier` is not relevant for CSRs and causes errors on newer (3.2) versions of OpenSSL.